lib/dark_worlds_server/runner_supervisor/runner.ex

```diff
@@ -266,7 +266,7 @@ defmodule DarkWorldsServer.RunnerSupervisor.Runner do
 
   @impl true
   def terminate(_reason, state) do
-    player_count = Enum.count(state.game_state.players) - state.bot_count
+    player_count = map_size(state.game_state.players) - state.bot_count
```

Comment:
is map_size mor efficient than Enum.count?

Reply:
Yes! Enum.count goes though the whole collection whole map_size Is calculated in constant time